### PR TITLE
Adzerk api update

### PR DESCRIPTION
**Pull request by tsegaran**
Updates Adzerk API to support active flights and flights for campaings

:eyeglasses: @dwick

### CODE REVIEW

File: reddit_adzerk/adzerk_api.py [hunk 1]
```diff
@@ -107,9 +107,9 @@ def _to_data(self):
         return {self._name: json.dumps(self._to_item())}
 
     @classmethod
-    def list(cls):
+    def list(cls, params = {}):
```
Comment (zeantsoi) [inline reddit_adzerk/adzerk_api.py]: Quick comment: mutable default args are a [common gotcha in Python](http://docs.python-guide.org/en/latest/writing/gotchas/).

Comment (tsegaran) [inline reddit_adzerk/adzerk_api.py]: Understood, but I'm not modifying it in that function... would you be more comfortable with **params ?

Comment (tsegaran) [inline reddit_adzerk/adzerk_api.py]: Actually I'll just change it to None, that's what the requests library does

Comment (zeantsoi) [inline reddit_adzerk/adzerk_api.py]: :thumbs_up:

File: reddit_adzerk/adzerk_api.py [hunk 2]
```diff
@@ -107,9 +107,9 @@ def _to_data(self):
         return {self._name: json.dumps(self._to_item())}
 
     @classmethod
-    def list(cls):
+    def list(cls, params = None):
```
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: nit: no spaces between `=` in kwargs

File: reddit_adzerk/adzerk_api.py [hunk 3]
```diff
@@ -281,6 +281,10 @@ def _from_item(cls, item):
                              for item in thing.CreativeMaps]
         return thing
 
+    @classmethod
+    def list(cls, is_active = False):
```
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: nit : spaces

File: reddit_adzerk/adzerk_api.py [hunk 4]
```diff
@@ -399,10 +403,10 @@ class Channel(Base):
     _name = 'channel'
     _fields = FieldSet(
         Field('Title'),
-        Field('Commission'), 
-        Field('Engine'), 
-        Field('Keywords'), 
-        Field('CPM'), 
+        Field('Commission'),
```
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: can this be a separate commit since it's an unrelated whitespace change?

File: reddit_adzerk/adzerk_api.py [hunk 5]
```diff
@@ -441,6 +445,15 @@ class Campaign(Base):
         Field('Price'),
     )
 
+    def fill_flights(self):
+        url = '/'.join([Campaign._base_url, self._name, str(self.Id), 'flight'])
+        print(url)
```
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: `print`

File: reddit_adzerk/adzerk_api.py [hunk 6]
```diff
@@ -441,6 +445,15 @@ class Campaign(Base):
         Field('Price'),
     )
 
+    def fill_flights(self):
```
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: what does this do exactly? i'm not following from the name and it's not used yet.  forgive me! :)

File: reddit_adzerk/adzerk_api.py (hunk 1 above)
Comment (dwick) [inline reddit_adzerk/adzerk_api.py]: @zeantsoi why are you here?!?! go have fun!

**Comment by dwick** [comment]
:nail_care: is this to be used somewhere else? the methods you've added don't have any invocations yet.

**Comment by dwick** [comment]
:fish: lgtm
